src/Diagnostics/Diagnostics.jl

```diff
@@ -14,7 +14,9 @@ export DiagnosticsGroup,
     setup_atmos_mass_energy_loss,
     setup_dump_state_diagnostics,
     setup_dump_aux_diagnostics,
-    setup_dump_spectra_diagnostics
+    setup_dump_spectra_diagnostics,
+    setup_dump_gcm_spectra_diagnostics
```

Comment:
This should be added in `setup_dump_spectra_diagnostics` itself. Will discuss how on Slack.